commons/ava_networks/test_gecko_network.go

```diff
@@ -133,7 +133,7 @@ func NewTestGeckoNetworkLoader(
 	// Defensive copy
 	serviceConfigsCopy := make(map[networks.ConfigurationID]TestGeckoNetworkServiceConfig)
 	for configId, configParams := range serviceConfigs {
-		if int(configId) >= bootNodeConfigIdStart && int(configId) < (bootNodeConfigIdStart + len(DefaultLocalNetGenesisConfig.Stakers)) {
+		if strings.HasPrefix(string(configId), bootNodeConfigIdPrefix) {
 			return nil, stacktrace.NewError("Config ID %v cannot be used as it's being used as a boot node config ID", configId)
```

Comment:
This log message isn't actually accurate - the reason we're rejecting them is because they share a prefix (even if that actual config ID might be free). Can we:
1) adjust this to reflect why their config actually got rejected
2) give them remediation steps (which in this case means not using that prefix, which means they need to know what prefix was disallowed)

Reply:
And actually, I missed it the first time around but the service IDs have the same problem - can we fix that too?